verifier/src/main/kotlin/tech/figure/classification/asset/verifier/config/VerifierEvent.kt

```diff
@@ -45,7 +45,7 @@ sealed interface VerifierEvent {
      * @param block The [BlockData] encountered by the stream processor.  Contains raw Provenance Blockchain
      * information.
      */
-    data class NewBlockReceived internal constructor(val block: BlockData) : VerifierEvent
+    data class NewBlockReceived internal constructor(val blockHeight: Long) : VerifierEvent
```

Comment:
These events are actually two-parts, so there should be another `NewBlockReceived` class in the class below this.  So that, along with the comments on both about the `BlockData` will also need to be modified

Reply:
Updated the comments. was there anything else you were wanting done here?